hyperspy/drawing/signal1d.py

```diff
@@ -282,28 +294,31 @@ def color(self, color):
             plt.setp(self.line, **self.line_properties)
             self.ax.figure.canvas.draw_idle()
 
+    def _get_pointer_text(self):
+        return self.hspy_figure._get_pointer_text(pointer=self.pointer)
+
     def plot(self, data=1):
         f = self.data_function
         if self.get_complex is False:
-            data = f(axes_manager=self.axes_manager).real
+            data = f(self.axes_manager, self.resizable_pointer).real
         else:
-            data = f(axes_manager=self.axes_manager).imag
+            data = f(self.axes_manager, self.resizable_pointer).imag
         if self.line is not None:
             self.line.remove()
         self.line, = self.ax.plot(self.axis.axis, data,
                                   **self.line_properties)
         self.line.set_animated(self.ax.figure.canvas.supports_blit)
-        self.axes_manager.events.indices_changed.connect(self.update, [])
-        self.events.closed.connect(
-            lambda: self.axes_manager.events.indices_changed.disconnect(
-                self.update), [])
+#        self.axes_manager.events.indices_changed.connect(self.update, [])
```

Comment:
I have commented this, because in `Signal1DFigure`, the line(s) is/are already been updated when indices of the `axes_manager` are changed. This connection is then not necessary and it was actually redrawing twice the same thing. I think, that this makes sense but I would prefer to make sure that I haven't miss something!